jets/e/secp.c

```diff
@@ -0,0 +1,356 @@
+/* j/5/secp.c
+**
+*/
+#include "all.h"
+#include "../include/secp256k1.h"
+#include "../include/secp256k1_recovery.h"
+
+// end-to-end test:
+//   =hash 123
+//   =priv 99
+//   =nonce (make-k:secp256k1:secp:crypto `@uvI`hash priv)
+//   =sig (ecdsa-raw-sign:secp256k1:secp:crypto `@uvI`hash priv)
+//   =reco (ecdsa-raw-recover:secp256k1:secp:crypto `@uvI`hash v:sig  r:sig s:sig)
+
+
+
+
+// util funcs
+//
```

Comment:
I don't think we use multi-line `//` for section titles anywhere, you'll want to use `/*` instead. Same for `// sign hash` and `// recover pubkey`.

```
/* util funcs
*/
```

There's also a bunch of newlines above this comment, one is plenty.

Reply:
done